packages/cli/src/commands/migrate.ts

```diff
@@ -40,16 +39,11 @@ type Context = {
   sourceFiles: Array<string>;
 };
 
-type ParsedModuleResult = {
-  edgeList: Array<[string, string | undefined]>;
-  coreDepList: Array<string>;
-};
-
 migrateCommand
   .name('migrate')
   .description('Migrate Javascript project to Typescript')
   .requiredOption('-r, --root <project root>', 'Base dir (root) of your project.')
-  .requiredOption('-e, --entrypoint <entrypoint>', 'entrypoint js file for your project')
+  .option('-e, --entrypoint <entrypoint>', 'entrypoint js file for your project')
```

Comment:
Dependency cruiser will glob and construct many graphs in the case we have multiple entrypoints.

If we constrain to an entrypoint it will only be applicable for libraries or partial migrations. In an ember use case, we will look at app, lib, etc for these files. So it won't be appropriate.

Reply:
Discussed in-person. For performance reasons we may want to make this required, but for ember applications this is not necessary since there isn't a single entrypoint to construct a graph from.